tests/installation/logpackages.pm

```diff
@@ -28,7 +37,14 @@ sub run {
     else {
         assert_screen 'before-package-selection', 300;
     }
-    select_console 'install-shell';
+    if (check_var('BACKEND', 'ipmi')) {
+	select_console('root-ssh');
+	$serialdev = 'sshserial';
+	set_var('SERIALDEV', 'sshserial');
+    }
+    else {	
+       select_console 'install-shell';
+    }
```

Comment:
I think the console should stay install-shell but if necessary change that console depending on the backend in lib/susedistribution.

Reply:
Got it, thank you for your great suggestion and comments.